pkg/gatherers/clusterconfig/scheduler.go

```diff
@@ -0,0 +1,88 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/insights-operator/pkg/gatherers/common"
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+// GatherSchedulers collects information about schedulers
+//
+// * Location in archive: config/pod/openshift-authentication/logs/{pod-name}/errors.log
+// * Since versions:
+//   * 4.10+
+func (g *Gatherer) GatherSchedulers(ctx context.Context) ([]record.Record, []error) {
+	gatherConfigClient, err := configv1client.NewForConfig(g.gatherKubeConfig)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	return gatherSchedulerInfo(ctx, gatherConfigClient)
+}
+
+func gatherSchedulerInfo(
+	ctx context.Context, configClient configv1client.ConfigV1Interface,
+) ([]record.Record, []error) {
+	schedulers, err := configClient.Schedulers().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	var records []record.Record
+	for i := range schedulers.Items {
+		scheduler := &schedulers.Items[i]
+
+		records = append(records, record.Record{
+			Name: fmt.Sprintf("config/schedulers/%v/%v", scheduler.Namespace, scheduler.Name),
```

Comment:
It looks like the `scheduler` is cluster-wide (i.e no namespace) and this leads to no name directory.